code/code/cmd/cmd_deathstroke.cc

```diff
@@ -62,69 +62,113 @@ static int deathstroke(TBeing *caster, TBeing *victim)
 
   int dam = caster->getSkillDam(victim, SKILL_DEATHSTROKE, lev, caster->getAdvLearning(SKILL_DEATHSTROKE));
 
-  if ((caster->bSuccess(bKnown+ percent, SKILL_DEATHSTROKE) &&
-         (i = caster->specialAttack(victim,SKILL_DEATHSTROKE)) && 
-         (i != GUARANTEED_FAILURE)) ||
-      !victim->awake()) {
+  if (caster->bSuccess(bKnown+percent, SKILL_DEATHSTROKE) || !victim->awake()) {
     wasSuccess = TRUE;
+
     if (!dam) {
```

Comment:
How would this happen?

Reply:
I didn't dig into how they're calculating skill damage deep enough to figure that out, but it seems like there are times when you succeed but the damage is non-existent.  A lot of stuff goes through getSkillDam() and so I didn't really mess with it at this point, but it's odd behavior if you ask me to have a skill succeed and somehow the damage is still 0.